### PR TITLE
Command args

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Improve commandline argument handling.

## How was this change tested?
Locally.

## Which documentation and/or configurations were updated?
README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,18 +135,29 @@ Note that the druids are unique.
 
 ### Seed the cache
 ```
-bin/generate-cache 100000 false false
+$ bin/generate-cache -h
+Usage: bin/generate-cache [options]
+    -s, --sample SAMPLE              Sample size, otherwise all druids.
+    -o, --overwrite
```
Comment (mjgiarlo) [inline README.md:141]: Worth adding documenting to the `-o` and `-r` args?

File: README.md [hunk 2]
```diff
@@ -167,13 +178,17 @@ Note that the validation is parallelized, so it is much faster than the other pr
 
 ### Validate roundtrip mapping (to Cocina from Fedora then to Fedora from Cocina)
 ```
-bin/validate-cocina-roundtrip 100 true
+$ bin/validate-cocina-roundtrip -h
+Usage: bin/generate-cache [options]
```
Comment (mjgiarlo) [inline README.md:182]: I think this may be a copypasta typo. (`generate-cache` should be `validate-cocina-roundtrip`?)

File: README.md [hunk 3]
```diff
@@ -167,13 +178,17 @@ Note that the validation is parallelized, so it is much faster than the other pr
...
+    -s, --sample SAMPLE              Sample size, otherwise all druids.
+    -r, --random
```
Comment (mjgiarlo) [inline README.md:184]: Worth adding documentation?

**Comment by justinlittman** [comment]
Thanks @mjgiarlo . I made the suggested changes.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**